cmd/kube-proxy/app/server_test.go

```diff
@@ -268,23 +292,36 @@ nodePortAddresses:
 
 		options := NewOptions()
 
-		yaml := fmt.Sprintf(
+		baseYAML := fmt.Sprintf(
```

Comment:
May be better to use Go templates rather than `fmt.Sprintf` so we don't have to remember the order of parameters in a big format string.

Reply:
I will make this part of the cleanup efforts in a separate PR, as discussed.